api/api_client.py

```diff
@@ -88,6 +89,7 @@ def api_init(kube_config_file=None, host=None, token_filename=None, cert_filenam
             config.load_kube_config(config_file=kubeconfig_path, context=context, client_configuration=configuration)
 
         api_client = ApiClient(configuration=configuration)
+        api_version = client.VersionApi(api_client=api_client)
```

Comment:
Maybe we should move it to the CVE scan function ? Because we use the version only in the scan function.